server/neptune/workflows/activities/command/subcommand_test.go

```diff
@@ -1,35 +1,35 @@
-package terraform_test
+package command_test
 
 import (
 	"testing"
 
-	"github.com/runatlantis/atlantis/server/neptune/workflows/activities/terraform"
+	"github.com/runatlantis/atlantis/server/neptune/workflows/activities/command"
 	"github.com/stretchr/testify/assert"
 )
 
 func TestCommandArguments_Build(t *testing.T) {
 	t.Run("with flags", func(t *testing.T) {
-		c := terraform.NewSubCommand(terraform.Show)
```

Comment:
`command.Show` doesn't read as intuitively as `terraform.Show` imo.

Reply:
What if we did `command.TerraformShow`?